### ISSUE TITLE
Deprecation warning using python 3.7

### ISSUE THREAD

**Issue by MerlinPendragon** [skuschel/postpic#217]
Dear developers, I got a deprecation warning every time when importing postpic. It seems to be related to python version. Here are the details:

```python
import postpic as pp
print(pp.__version__)

'0+unknown'
```

```python
import sys
print(sys.version)

3.7.3 (default, Mar 27 2019, 22:11:17) 
[GCC 7.3.0]
```

```
/***/postpic-master/postpic/particles/scalarproperties.py:99: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  class ScalarPropertyContext(collections.Mapping):
/***/postpic-master/postpic/datareader/datareader.py:298: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  class Simulationreader_ifc(collections.Sequence):
```

**Comment by skuschel** [comment]
Dear @MerlinPendragon 
thank you for your feedback! We really need update this before python 3.8 is released, but till then this does not affect functionality in any way.
We hardly get any feedback from users, so if there is more on your mind, please do not hesitate to use the issue tracker for questions or contributions.
Thank you
Stephan

**Comment by MerlinPendragon** [comment]
Dear Stephan,

Thank you for your timely response! I just started to use this powerful tool, but I have known this repo a while ago. Since I've just started a new project, I thought I should give it a try. 

You did exactly what I dreamed. I've always wanted to organize my post-processing scripts into a package. Thanks for your hard work. This tool is very useful. I hope I could contribute much more in the future.

Cheers
Xiabing Li
from Peking University
